src/untangled/client/impl/application.cljc

```diff
@@ -218,19 +218,21 @@
                                       (swap! initial-state assoc :ui/locale "en-US")
                                       initial-state)
                                     (assoc initial-state :ui/locale "en-US"))
-        config                    {:state       initial-state-with-locale
-                                   :send        (fn [tx cb]
-                                                  (server-send (assoc app :reconciler @rec-atom) tx cb))
-                                   :migrate     (or migrate tempid-migrate)
-                                   :normalize   true
-                                   :remotes     remotes
-                                   :pathopt     pathopt
-                                   :merge-ident (fn [reconciler app-state ident props]
-                                                  (update-in app-state ident (comp sweep-one merge) props))
-                                   :merge-tree  (fn [target source]
-                                                  (merge-handler mutation-merge target source))
-                                   :parser      parser
-                                   :shared      shared}
+        reconciler-options        (update reconciler-options :migrate #(or % tempid-migrate))
+        config                    (merge {:state       initial-state-with-locale
+                                          :send        (fn [tx cb]
+                                                         (server-send (assoc app :reconciler @rec-atom) tx cb))
+                                          :migrate     nil
+                                          :normalize   true
+                                          :remotes     remotes
+                                          :pathopt     true
+                                          :merge-ident (fn [reconciler app-state ident props]
+                                                         (update-in app-state ident (comp sweep-one merge) props))
+                                          :merge-tree  (fn [target source]
+                                                         (merge-handler mutation-merge target source))
+                                          :parser      parser
+                                          :shared      nil}
+                                    reconciler-options)
```

Comment:
So, in principle I agree with you. In practice I don't want to debug people's mess on slack channels and in issues when they screw up the main thing Untangled does for them.  So, for example, if they specify `:remotes [:a]` and give `:networking (make-network)` it is likely to not work right. If we're going to expose those kinds of potential problems, then there needs to be a bunch of documentation telling them how to use the additional options that could conflict with how Untangled works. E.g. "If you want to override merge-tree, you need to make sure to do a sweep step on the not-found markers"...and the explanation that goes with that.